app/views/alterations_amends.scala.html

```diff
@@ -28,55 +29,52 @@
     @reference(requestObject.getPageTitle)
     <p class="font-xsmall"><a href="@ersUtil.getPageBackLink(schemeId, ersUtil.PAGE_ALT_AMENDS)" class="link-back">@Html(messages("ers.back"))</a></p>
 
-    @header(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "question"))
     @if(flash.get("alt-amends-not-selected-error").isDefined) {
-    <div id="errors" class="validation-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
-        <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
-            @Html(messages("validation.summary.heading"))
-        </h1>
-        <ul class="validation-summary-message">
-            <li class=validation-summary-message"><a href="#error-summary" data-focuses="error-summary">@Html(ersUtil.getPageElement(schemeId, ersUtil.PAGE_ALT_AMENDS, "err.message"))</a></li>
-        </ul>
-    </div>
+        <div class="flash error-summary error-summary--show " id="error-summary-display" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@messages("validation.summary.heading")</h2>
```

Comment:
Should this h2 have class "h3-heading"?

Reply:
the play ui helper functions creates headings in this way, I did this to keep the consistency across the service

Reply:
Cool, that's fine then